packages/fxa-settings/src/components/AlertBar/index.tsx

```diff
@@ -3,46 +3,68 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React, { ReactNode } from 'react';
+import { createPortal } from 'react-dom';
 import { useEscKeydownEffect, useChangeFocusEffect } from '../../lib/hooks';
 import { ReactComponent as CloseIcon } from 'fxa-react/images/close.svg';
+import Portal from 'fxa-react/components/Portal';
 
 type AlertBarProps = {
-  children: ReactNode | string;
+  children: ReactNode;
   onDismiss: Function;
 };
 
 export const AlertBar = ({ children, onDismiss }: AlertBarProps) => {
+  const alertBarRoot = document.getElementById("alert-bar-root");
```

Comment:
Eh so, I converted this back to a draft while I check on something here - while `document.getElementById` isn't ideal in React it seemed preferable to creating an element ref in `AppLayout` and passing it into every `AlertBar` instance and ref forwarding wasn't doing what I wanted, but going to look into one more idea...